tests/v2/test_0024-use-regular-array.py

```diff
@@ -52,9 +52,14 @@ def test_empty_array_slice():
     )
     assert to_list(listarray[2, 1, []]) == []
     assert listarray.typetracer[2, 1, []].form == listarray[2, 1, []].form
+<<<<<<< HEAD
+    assert to_list(listarray[2, [1], []]) == []
+    assert listarray.typetracer[2, [1], []].form == listarray[2, [1], []].form
+=======
     assert to_list(listarray[2, [1], []]) == [[]]
     # FIXME
     # assert listarray.typetracer[2, [1], []].form == listarray[2, [1], []].form
+>>>>>>> 0c477f49cdcb3ed31fb70655391633b007e739fb
```

Comment:
@ioanaif what did you want to do about this test? Is it broken for another reason, and therefore you're commenting it out here? Maybe we should skip it instead of comment it out, so that we have a high-level indication it's not working?

```suggestion
assert to_list(listarray[2, [1], []]) == [[]]
# FIXME
# assert listarray.typetracer[2, [1], []].form == listarray[2, [1], []].form
```

Reply:
@agoose77 It got in there by mistake while the PR was still a draft; I reverted all the changes to that test before marking the PR as ready

Reply:
Oh, strange. Maybe my client didn't refresh! Ignore me :smile: